Automattic/wp-calypso
# Marketplace: Persist pricing selector options between navigation and page reloads

#### Changes proposed in this Pull Request

* This pull requests persist in the state of the pricing selector options. 

#### Testing instructions



* Go to wordpress.com/plugins/
* It should show monthly as the default pricing option selected. 
* Switch to annual price option
* Click on any paid plugin
* Verify if the pricing selector shows the annual option selected by default. 
* Verify if it still persists the state across page reloads and navigation.
* Verify same for the individual site (wordpress.com/plugins/[SITE_ID] view as well. 

Also, check for potential regressions!



Related to #57102
